inyoka_theme_default/templates/portal/privmsg/index.html

```diff
@@ -0,0 +1,209 @@
+{#
+  portal/privmsg/index.html
+  ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Display a list of private messages.
+
+  :copyright: (c) 2007-2015 by the Inyoka Team, see AUTHORS for more details.
+  :license: BSD, see LICENSE for more details.
+#}
+
+{% import 'macros.html' as macros %}
+
+{% extends 'portal/privmsg/base.html' %}
+
+{% block breadcrumb %}
+  {{ macros.breadcrumb_item(_('Portal'), href('portal', 'privmsg', 'inbox')) }}
```

Comment:
hm, dont know if the label „Portal“ is the best one here. Personally, i would prefer `{{ super() }}` and something like `{{ macros.breadcrumb_item(_('Inbox'), href('portal', 'privmsg', 'inbox')) }}` afterwards. (With the power of `super()` "Portal" should be added before the new Inbox as privmsg/index.html extends portal/base.html via privmsg/base.html)